TodoApp/Src/TodoApp.Api/App_Start/JsonFormatterConfig.cs

```diff
@@ -9,7 +9,6 @@ public static class JsonFormatterConfig
     {
         public static void Register(HttpConfiguration config)
         {
-            // Configure Json Formatter
             JsonMediaTypeFormatter jsonFormatter = config.Formatters.JsonFormatter;
             jsonFormatter.SupportedMediaTypes.Add(new MediaTypeHeaderValue("text/html"));
```

Comment:
It is quite a nasty hack, you have here. I don't need you to remove it, but it surely deserves at least a comment 😉